gcloud/bigquery/client.py

```diff
@@ -95,7 +96,6 @@ def dataset(self, name):
         :param name: Name of the dataset.
 
         :rtype: :class:`gcloud.bigquery.dataset.Dataset`
-        :returns: a new ``Dataset`` instance
```

Comment:
Your comment pointed out that the element was redundant:  it added no new information to the docstring beyond the summary line and `:rtype:`.  I will rebase out 35697d49ea3a0ed76c565f7ecb68b4c8394ff196